python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -281,18 +280,20 @@ def _func_build_rf(
         )
 
     @staticmethod
-    def _fit(model, X_df_list, y_df_list,
-             convert_dtype, r):
-        if len(X_df_list) != len(y_df_list):
-            raise ValueError("X (%d) and y (%d) partition list sizes unequal" %
-                             len(X_df_list), len(y_df_list))
-        if len(X_df_list) == 1:
-            X_df = X_df_list[0]
-            y_df = y_df_list[0]
+    def _fit(model, input_data, convert_dtype):
+        X = input_data[0][0]
+        y = input_data[0][1]
```

Comment:
Also, I think `cuml.dask.input_utils.concatenate` simplifies this pattern, since it automatically selects between cupy and dask_cudf. So just:
```
X = input_utils.concatenate([item[0] for item in input_data])
y =  input_utils.concatenate([item[1] for item in input_data])
```
with no conditional based on datatype should work

Reply:
I had looked into the input_utils.concatenate function, the function can concatenate cudf Dataframes and cupy arrays but not cudf Series. I will update the concatenate function to include  the option to concatenate Series as well and then use this function.